.github/ISSUE_TEMPLATE/feature-enhancement-request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Feature enhancement request
+about: Suggest an idea for this project
+
+---
+
+**Is your feature request related to a problem? Please describe**
```

Comment:
Do we need this section plus the desired solution section? Can we just have 1? If it's related to a problem, shouldn't they just file a bug report?

Reply:
I had that in there in case is the RFE was forked from a bug. We can just put that in the comments.